dell/xps/15-7590/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+  ];
+
+  # TODO: boot loader
+  boot.loader.systemd-boot.enable = lib.mkDefault true;
```

Comment:
I don't think we should hard-code to systemd-boot here. Some people might prefer grub for different reasons.